src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -162,6 +162,8 @@
         <KeyBinding Key="F5"
                     Command="{Binding Path=DataContext.HomeSpaceViewModel.RunSettingsViewModel.RunExpressionCommand, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}"
                     CommandParameter="{Binding Path=DataContext.HomeSpaceViewModel.RunSettingsViewModel.RunInDebug, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}" />
+        <MouseBinding Gesture="Shift+LeftDoubleClick"
+                      Command="{Binding Path=DataContext.HomeSpaceViewModel.ShowInCanvasSearchCommand, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}" />
```

Comment:
HomeSpaceViewModel is `WorkspaceViewModel`. Model for `WorkspaceViewModel` is `WorkspaceModel`. `CustomNodeWorkspaceModel` inherited from `WorkspaceModel`. So, it will work on both custom workspace and home workspace, tried it manually.